torchao/quantization/pt2e/quantizer/arm_inductor_quantizer.py

```diff
@@ -34,11 +45,16 @@
     QuantizationAnnotation,
     QuantizationSpec,
 )
+from torchao.utils import TORCH_VERSION_AT_LEAST_2_7
 
 from .x86_inductor_quantizer import (
     X86InductorQuantizer,
 )
 
+if TORCH_VERSION_AT_LEAST_2_7:
+    torch._inductor.config.pre_grad_custom_pass = quant_lift_up
```

Comment:
what happens when multiple backend set this one? https://github.com/pytorch/ao/blob/7690612a018fea6c0ed4efeaa4809c0798d12786/torchao/quantization/pt2e/quantizer/x86_inductor_quantizer.py#L1640